pywinauto/macos/macos_functions.py

```diff
@@ -42,6 +42,7 @@
 import subprocess
 import os
```

Comment:
Built-in modules should be imported first: os, subprocess. Then 3-rd party modules (six, all PyObjC stuff), then project specific modules. Grouped imports should be separated by empty line.